sensor_validate.py

```diff
@@ -1,19 +1,26 @@
+spike_threshold_limit         = {
+                                  'soc'      :  0.05 ,
+                                  'current'  :  0.1  ,
+                                }
 
-def  _give_me_a_good_name(value, nextValue, maxDelta):
+def isAbnormalSpike(value, nextValue, maxDelta):
```

Comment:
Thanks. I changed the code in such a way that I retain existing method name isAbnormalSpike. but now, it will return True for True case (when we get abnormal spike), instead of vice versa which was happening before.